### PR TITLE
chore(docs): Added guide on remote access

### PR DESCRIPTION
**Pull request by thariq-shanavas**
A brief guide on different ways to access immich remotely. I use Wireguard to connect to my server - apologies for any technical mistakes in what I've written about reverse proxy. Let me know and I'll be happy to fix it!

### CODE REVIEW

File: docs/docs/guides/remote-access.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- Tailscale needs to be installed and running on both server-side and client-side.
+
+## Option 3: Self-signed certificate
+Using a [self-signed certificate](https://en.wikipedia.org/wiki/Self-signed_certificate) protects you against man-in-the-middle attacks, but exposes your Immich instance to the web.
+
+### Pros
+- No additional software needs to be installed client-side
+
+### Cons
+- Both the Immich web interface and API may be exposed to the internet. Immich is under very active developement and the existence of severe security vulnerabilities cannot be ruled out.
```
Comment (bo0tzz) [inline docs/docs/guides/remote-access.md:45]: A self-signed cert is not technically a standalone remote access option, but would be a part of a remote proxy setup. I would suggest removing this section and instead adding a note to the reverse proxy section, indicating that with a remote proxy a certificate should be used for https. The options there are to get a free certificate via Let's Encrypt (preferred), or to use a self signed one.

File: docs/docs/guides/remote-access.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Option 4: Reverse Proxy
+A reverse proxy is a service that sits between web servers and clients. A reverse proxy increases security by hiding the server IP address, which makes targeted attacks like [DDoS](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/) harder.
```
Comment (bo0tzz) [inline docs/docs/guides/remote-access.md:48]: A reverse proxy does not necessarily hide the server IP/make targeted attacks harder. This is only the case if you're using something like cloudflare, but not when you're running a reverse proxy like nginx on your server. Maybe cloudflare and a local proxy server should be separate sections?

**Review by bo0tzz — CHANGES_REQUESTED** [review]
This is pretty much exactly what I had in mind, thank you!

File: docs/docs/guides/remote-access.md (hunk 2 above)
Comment (thariq-shanavas) [reply docs/docs/guides/remote-access.md]: Thank you, that clarifies some lingering confusion I had about reverse proxies. I made the changes.

**Review by bo0tzz — APPROVED** [review]

**Comment by jrasm91** [comment]
This looks pretty good. Any thoughts on making the pro's and con's list start with ✅ and ❌ or similar?

**Comment by thariq-shanavas** [comment]
Thank you for the formatting fix! I was just about to get to it.

**Pull request merged**